VTEX - GiftCard Hub API.json

```diff
@@ -2273,7 +2341,7 @@
                 }
             },
             "Item": {
```

Comment:
This component is used in a request body, therefore it must abide by the OAS guidelines for request bodies, please include examples to each field.

See our [openapi template](https://github.com/vtex/openapi-schemas/blob/master/templates/VTEX%20-%20Template%20openAPI.jsonc)